index.html

```diff
@@ -28,7 +28,7 @@
           </li>
         </ul>
       </div>
-      <button type="button" class="hamburger-icon">
+      <button type="button" class="hamburger-icon" aria-label="Open menu">
         <span class="bar"></span>
         <span class="bar"></span>
         <span class="bar"></span>
```

Comment:
- [ ] Nice work so far but you are almost there, I'm not able to inline your line `42-51`, As required for this project your tab must focus on each block on your project but I noticed that it's only focusing on links, icons, and buttons. Headings and other texts should be focusable too. Kindly fix this so that your project would be accessible and match all the requirements for this milestone. This is a short video showing how it's not focusing on texts:

https://user-images.githubusercontent.com/88894525/221993136-ae78c118-e38b-49cb-b084-46fc3e99a872.mp4

_I even see that there's not focus on your logo we can't access on it , kindly make all your website accessible :wink: _

Reply:
@Alfredbis29 Thank you very much!  Your feedback is amazing because you are showing me with a video and that makes it easy to understand. Have a very good day.